### PR TITLE
Cleanup and document running sample sekaid app

### PR DESCRIPTION
**Pull request by ethanfrey**
This is getting `sekaid` into a state people can run it, and document how to do so

- [x] document how to create keys
- [x] set up sample genesis config
- [x] document how to run a node with default config
- [x] fix minor app.go misconfigurations
- [ ] document how to use running node
- [ ] document how to configure node with different keys

It is mostly finished except for documenting how to customize. And examples of sending tx

### CODE REVIEW

**Comment by asmodat** [comment]
noticed that version flag is missing from `sekaicli` and results in error, it is useful for automation and debug

sekaid version returns empty string

**Comment by asmodat** [comment]
`sekaicli keys add -i <name>` generates keys with prefix `cosmos` instead of `kira`

example:


```
sekaicli keys add -i test
Enter a passphrase to encrypt your key to disk:
Repeat the passphrase:
> Enter your bip39 mnemonic, or hit enter to generate one.
****************************************************************
> Enter your bip39 passphrase. This is combined with the mnemonic to derive the seed. Most users should just hit enter to use the default, ""


- name: test
  type: local
  address: cosmos10ryzw3lek4je2dzpvwm9jd3x696jj4us5ve9k8
  pubkey: cosmospub1addwnpepq2fkqzzauc96qdj7t4vr2ds0mqf40yuaghafp7k739qxx058937u2wp48r7
  mnemonic: ""
  threshold: 0
  pubkeys: []
```

File: SAMPLE_APP.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+sekaicli keys mnemonic
+# For quickstart (insecure), use this mnemonic instead of your locally generated secure one:
+# echo that remain online near smooth snack general regular replace brisk ahead seek online awake portion essay spare pyramid squeeze among invest group fire
+sekaicli keys add -i demo
+# choose a passphrase to encrypt, enter mnemonic, and then 
+sekaicli keys add -i admin --account 2
+# use the same values as above, just a different BIP44 derivation path
+```
+
+If you used the default (insecure) mnemonic, `sekai keys list` should now show the following.
```
Comment (asmodat) [inline SAMPLE_APP.md]: `sekaicli keys list` not `sekai keys list`

**Comment by asmodat** [comment]
looks like it is not possible to add memo to the tx as well as --fees flag is not found

trying to send transaction with the same amount twice results in the following error (which is probably an expected and desired output because we do not want people sending same tx 2x by mistake)
```
ERROR: broadcast_tx_sync: response error: RPC error -32603 - Internal error: Tx already exists in cache
```
However, I tried to go around this by setting memo but it is not possible to do.

I think there should be an additional flag called "seed" that can be used to propagate the same tx many times by changing it.

File: Essential-Toolkit.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Basic Environment Setup
+
+_Note: Do NOT use in production, this is a quick setup example._
+
+## Minimum Requirements
+
+> Ubuntu 18.04 LTS
+>
+> Instance Type: t3a.small
```
Comment (etaeh) [inline Essential-Toolkit.md]: Why os and aws instance type is defined here?

**Comment by ethanfrey** [comment]
> sekaicli keys add -i <name> generates keys with prefix cosmos instead of kira

I saw that too, made another issue for it. This PR is just to get a runable binary.
Merge early and merge often.

See https://github.com/kira-ex/sekai/issues/6 (and please add issues for any other missing functionality)

**Comment by ethanfrey** [comment]
> trying to send transaction with the same amount twice results in the following error (which is probably an expected and desired output because we do not want people sending same tx 2x by mistake)

We have same cli as gaia - please check there and open unique issue for this.

I believe your issue is running them too close together, as it queries the nonce from the chain, and if you submit the same tx with the same nonce and same signer it will be the same bytes - which is rejected early by the mempool with the above message

**Comment by ethanfrey** [comment]
> sekaid version returns empty string

This now works for both `sekiad` and `sekaicli`

File: Essential-Toolkit.md (hunk 2 above)
Comment (ethanfrey) [reply Essential-Toolkit.md]: I agree, remove this, and request **4+GB** ram and 2+ cpus

No need to skimp so hard that random commands fail. And digital ocean is cheaper if you want to save there

Comment (ethanfrey) [reply Essential-Toolkit.md]: I am fine with the OS being specified, so all following commands "just work"

Comment (ethanfrey) [reply Essential-Toolkit.md]: btw, this seems like dev machine requirements, likely a laptop.

File: README.md [hunk 3]
```diff
@@ -1 +1,67 @@
...
+
+> Create or cleanup the destination folder
+
+```
+mkdir -p $GOPATH/src/github.com/kira-ex && \
+ cd $GOPATH/src/github.com/kira-ex && \
+ rm -rfv  ./sekai
+```
+
+> Clone seki repository, you can change branch or tak by defining `tag` variable
```
Comment (ethanfrey) [inline README.md]: s/tak/tag/

**Pull request merged**